deploy.tex

```diff
@@ -1,50 +1,48 @@
 \section{Continuous Deployment across the Rubin facilities} \label{sec:deploy}
 
-Being cloud ready is certainly the biggest challenge for most projects/applications which intend to leverage the commercial cloud.
-Early on we adopted Kubernetes which provided a service architectures that is well isolated from the underlying infrastructure.
+Being cloud-ready is certainly the biggest challenge for most projects/applications which intend to leverage the commercial cloud.
```

Comment:
hyphenation a real divisive device I usually avoid completely, I suppose in US it is more popular.